test/widgets/test_thermal_zone.py

```diff
@@ -0,0 +1,14 @@
+import os.path
+from libqtile import widget
+
+def test_thermal_zone_getting_value():
+    # If thermal zone zero exists - get value
+    zone_zero = '/sys/class/thermal/thermal_zone0/temp'
+    zone_exists = os.path.isfile(zone_zero)
+    
+    if zone_exists:
+        thermal_zone = widget.ThermalZone(zone='0')
+        output = thermal_zone.poll()
+        print(output)
```

Comment:
Remove this line. Presumably you had it for your own debugging.

Reply:
Done